.rubocop.yml

```diff
@@ -1,6 +1,7 @@
 inherit_from: .rubocop_todo.yml
 
 AllCops:
+  NewCops: enable
```

Comment:
FYI: I decided to enable new cops to suppress rubocop warnings. Looks like all works fine except `Lint/DuplicateBranch` rule (it found copy/paste in code and throw warnings), so I decided to disable this one.

Reply:
:+1: